src/activities/guesscount/Guesscount.qml

```diff
@@ -0,0 +1,462 @@
+            }
+            Rectangle{
+                width:parent.width/2
+                height:100
+                color:"transparent"
+                Rectangle {
+                    anchors.horizontalCenter: parent.horizontalCenter
+                    width:400;
+                    height: 100;
+                    radius: 20.0;
+                    color: "#024c1c"
+                    GCText{
+                        anchors.horizontalCenter: parent.horizontalCenter
+                        fontSize: largeSize
+                        text: qsTr("Numbers")
+                    }
+                }
+            }
+
+            Repeater{
+                model: items.data
+                Item{
+                    id:root2
+                    width:100
+                    height:100
+                    MouseArea{
+                        id:mousearea
+                        //anchors.fill:rec2
+                        width: 100
+                        height: 100
+                        property Rectangle obj: rec2
+                        drag.target: rec2
+                        onReleased: {
+                            parent = rec2.Drag.target !== null ? rec2.Drag.target : root2
+                            rec2.Drag.drop()
+                        }
+
+                        Rectangle {
+                            id:rec2
+                            width:100
+                            height: 100
+                            color:"green"
+                            property var datavalue: modelData
+                            Drag.active: mousearea.drag.active
+                            Drag.keys: [ "number" ]
+                            Drag.hotSpot.x: 50
+                            Drag.hotSpot.y: 50
+                            GCText{
+                                anchors.horizontalCenter: parent.horizontalCenter
+                                anchors.verticalCenter: parent.verticalCenter
+                                text:Number(modelData).toString()
+                                fontSize: mediumSize
+                            }
+
+                            radius: 20.0
+                            states: State {
+                                when: mousearea.drag.active
+                                ParentChange { target: rec2; parent: root2 }
+                                AnchorChanges { target: rec2; anchors.verticalCenter: root2.anchors.verticalCenter; anchors.horizontalCenter: root2.anchors.horizontalCenter }
+                            }
+                        }
+                    }
+                }
+            }
+
+        }
+
+        // operation row
+        Row{
+            id:row4
+            spacing: 50
+            anchors{
+                top:row3.bottom
+                topMargin:10
+                left:parent.left
+                leftMargin: 300
+            }
+            Repeater{
+                id:repeat
+                model:3
```

Comment:
how you do handle levels with more than 2 numbers and 1 operator?
I mean, for example, you have: 1 2 3 and need to find 6. A solution would be 1 + 2 + 3, is it handle?